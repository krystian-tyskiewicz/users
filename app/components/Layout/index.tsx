'use client'

import { ReactNode } from 'react'
import styled, { createGlobalStyle } from 'styled-components'

const GlobalStyle = createGlobalStyle`
  body {
    font-family: Helvetica, sans-serif;
    max-width: 960px;
    margin: 0 auto;
  }
`

const Wrapper = styled.main`
  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
  margin-top: 32px;
  padding: 16px;
  display: flex;
  flex-direction: column;
`

type Props = {
  children: ReactNode,
  className?: string
}

const Main = ({ children, className }: Props) => (
  <Wrapper className={className}>
    <GlobalStyle />
    {children}
  </Wrapper>
)

export default Main
