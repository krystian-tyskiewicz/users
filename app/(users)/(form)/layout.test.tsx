import { render } from '@testing-library/react'
import Layout from './layout'

it('renders correctly', () => {
  const { container } = render(<Layout>Test</Layout>)
  expect(container).toMatchSnapshot()
})
