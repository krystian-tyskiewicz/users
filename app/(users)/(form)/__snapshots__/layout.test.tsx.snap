// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <main
    class="sc-iCfKOy dgWaXE sc-ksdynI foZgKl"
  >
    Test
  </main>
</div>
`;
